lib/elocaps/ui/pages/history_page/game_card.dart

```diff
@@ -67,60 +81,87 @@ class GameCard extends HookConsumerWidget {
               Expanded(
                 child: Center(
                   child: Text(
-                    firstTeam
-                        .fold<int>(
-                            0,
-                            (previousValue, element) =>
-                                previousValue + element.score)
-                        .toString(),
+                    scoreToWinOrLose(firstTeam.fold<int>(
+                        0,
+                        (previousValue, element) =>
+                            previousValue + element.score)),
```

Comment:
You are using this calculation three times, can you refactor this calculation into a variable to ensure to do the calculation only once